app/models/article.rb

```diff
@@ -0,0 +1,16 @@
+class Article < ApplicationRecord
+  include PGEnum(status: %w[draft published trashed])
```

Comment:
Oh. no-no, we are not doing like this - https://betterprogramming.pub/how-to-use-enums-in-rails-6-87600e292476

Reply:
Thx! It is much simpler. Wonder why it didn't work for me the first time...